tic-tac-toe/script.js

```diff
@@ -0,0 +1,157 @@
+console.log ("It Runs");
+
+var board = document.getElementById("board");	
+
+console.log(board);
+
+var buttons = document.getElementsByClassName("grid");
+console.log(buttons);
+
+// buttons.addEventListener("click", function(userClick) {
+// 	// for (var i = 0; i < buttons.length; i++) 
+// 	userClick.preventDefault();
+	var input = buttons.value;
+//	console.log(input);
+
+	// body...
+//})
+
+var buttonOne = document.getElementById("one");
+var buttonTwo = document.getElementById("two");
+var buttonThree = document.getElementById("three");
+var buttonFour = document.getElementById("four");
+var buttonFive = document.getElementById("five");
+var buttonSix = document.getElementById("six");
+var buttonSeven = document.getElementById("seven");
+var buttonEight = document.getElementById("eight");
+var buttonNine = document.getElementById("nine");
+var resetButton = document.getElementById("clear");
+
+var counter = 0;
+var buttonArray = [];
+
+// for (var i = 0; i < buttons.length; i++) 
+
+// var clickButton = function(click) {
+// 	for (var i = 0; i < buttons.length; i++) 
+// 		counter ++;
+// 	}
+	// body...
+
+// VVV unsuccessful attempt at reset button below  VVV
+//  resetButton.addEventListener("click", function() {
+// 	document.getElementById("resetbutton").value = null;
+// });
+
+resetButton.addEventListener("click", function(click) {
+	buttonOne.innerHTML = ""; buttonOne.style.background = "default";
```

Comment:
Since you've already stored your buttons in an array, you can use a for loop here to keep your code DRY:
```
	for (button in buttons) {
		button.innerHTML = "";
	}
```